components/BusinessPage.vue

```diff
@@ -78,10 +78,11 @@
             class="business-list-container__result-count"
           >
             <p>
-              Unfortunately, there are currently no businesses in the Repair
-              Directory that match your search criteria.
+              {{ $t('noBusinesses') }}
+            </p>
+            <p>
+              {{ $t('widerRadius') }}
             </p>
-            <p>Try selecting a wider search radius.</p>
             <p>
               If you still don’t obtain results, it means that we don’t know of
```

Comment:
Not sure if this has/needs a translation?  "If you still don’t obtain results..."

Reply:
Sheesh, what I am thinking?  Yes, it has.